course_catalog/etl/youtube_test.py

```diff
@@ -173,17 +161,19 @@ def test_extract_with_bad_channel_id(
 def test_extract_with_unset_keys(settings):
     """Test youtube video ETL extract with no keys set"""
 
-    settings.VIDEOS_CONFIG_FILE_LOCATION = None
-    settings.YT_DEVELOPER_KEY = None
+    settings.YOUTUBE_CONFIG_FILE_LOCATION = None
+    settings.YOUTUBE_DEVELOPER_KEY = None
 
     assert list(youtube.extract()) == []
 
 
-@pytest.mark.usefixtures("mocked_channel_empty_response")
-def test_extract_with_no_channels(mocker, settings):
+@pytest.mark.usefixtures("video_settings")
+@pytest.mark.usefixtures("mocked_channel_response")
+@pytest.mark.parametrize("yaml_parser_response", [None, []])
+def test_extract_with_no_channels(mocker, yaml_parser_response):
     """Test youtube video ETL extract with no channels in data"""
-    settings.VIDEOS_CONFIG_FILE_LOCATION = "https://youtube_empty.yaml"
-    settings.YT_DEVELOPER_KEY = "key"
     mocker.patch("course_catalog.etl.youtube.get_youtube_client")
+    mock_yml_parser = mocker.patch("yaml.safe_load")
+    mock_yml_parser.return_value = yaml_parser_response
```

Comment:
Can simplify these both down to a single line `mocker.patch("yaml.safe_load", return_value=yaml_parser_response)`